Resources/ServerInfo/Guidebook/Service/FoodRecipes.xml

```diff
@@ -22,9 +22,15 @@ WARNING: This is not an automatically generated list, things here may become out
 - Chocolate = 6 Cocoa Powder, 2 Milk, 2 Sugar
 - Uncooked Animal Protein: Grind Raw Meat
 
+Buzz! Don't forget about Moth diet!
+- Cotton Dough = 5 Flour, 10 Fabric, 10 Water
+- Almost every other basic recipe also works with cotton dough (e.g. bread)
```

Comment:
this is not true since there's only recipes, either change or fix that please

Reply:
weh